qcow2reader_test.go

```diff
@@ -7,15 +7,18 @@ import (
 	"math/rand"
 	"os"
 	"path/filepath"
+	"slices"
```

Comment:
Not available in go 1.20?

Reply:
Replace with reflect.

Reply:
I think it is sensible to bump Go to 1.22

Reply:
Update in #39.